### PR TITLE
add some examples to documentation: cover, climate

### PR DESCRIPTION
**Pull request by nagyrobi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
               message: Hello {{ text }}
 ```
 
-in the event service call you can use any variable coming from the MQTT message besides the actual event
+In the event service call any variable coming from the MQTT message can be used between curly brackets. 
+
+### Example 1: cover control with state feedback
+![kép](https://user-images.githubusercontent.com/1550668/112142921-77daf580-8bd7-11eb-9626-ebfb3423629d.png)
```
Comment (dgomes) [inline README.md:60]: what's kép ?

File: README.md [hunk 2]
```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
...
+<br>UI theme set to `Hasp Light` in plate's web interface.
+
```
Comment (dgomes) [inline README.md:62]: we also need to tell users which HA entities are involved

File: README.md [hunk 3]
```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
...
+{"obj":"dropdown","id":4,"x":75,"y":235,"w":90,"h":30,"options":"off\nheat_cool\nheat\ncool\ndry\nfan_only"}
+{"obj":"btn","id":5,"x":68,"y":162,"w":25,"h":25,"toggle":false,"text":"-","text_font":28,"align":1}
+{"obj":"btn","id":6,"x":147,"y":162,"w":25,"h":25,"toggle":false,"text":"+","text_font":28,"align":1}
+{"obj":"label","id":7,"x":60,"y":120,"w":120,"h":30,"text":"Status","align":1,"padh":50}
+```
+**hasp-lvgl-custom-component config:**
+```yaml
+      - obj: "p2b3"
+        properties:
+          "val": "{{ state_attr('climate.thermostat_1','temperature') * 10 | int }}"
```
Comment (dgomes) [inline README.md:147]: I know your personal use case in which you need to multiply/divide, but we should explain this to users too, or remove altogether not to cause confusion

File: README.md (hunk 1 above)
Comment (nagyrobi) [reply README.md:60]: it's generated by the browser when I Ctrl+V the picture from clipboard

File: README.md (hunk 2 above)
Comment (nagyrobi) [reply README.md:62]: can be seen from the code, its self-explanatory

File: README.md (hunk 3 above)
Comment (nagyrobi) [reply README.md:147]: OK

File: README.md (hunk 2 above)
Comment (dgomes) [reply README.md:62]: you don't deal with end user too often, now do you 😄 ?

I also think it's better to name our entities in English (I'm also to blame in my example)

Comment (nagyrobi) [reply README.md:62]: I see what you mean, I already translated most of them to cover_1 and 2 but only one was left from my code, sorry.
I am also having a hard time reading others code but that makes me learn and understand better.

File: README.md (hunk 3 above)
Comment (nagyrobi) [reply README.md:147]: I wouldn't call this my own _personal use case_. In real life everyone needs to set thermostat in decimal values, at least in 0.5 degrees precision.

Comment (dgomes) [reply README.md:147]: Mine does not handle that precision 😓

Comment (nagyrobi) [reply README.md:147]: It's time for you to implement one :-)

Comment (nagyrobi) [reply README.md:147]: Modifications requested made.

Comment (dgomes) [reply README.md:147]: I read somewhere the Landon has a temperature sensor... waiting for it's support 😄

**Review by dgomes — APPROVED** [review]

**Pull request merged**